src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -470,6 +538,7 @@ ss::future<response_ptr> describe_configs_handler::handle(
               topic_property_compression,
               topic_config->properties.compression,
               request.data.include_synonyms,
+              make_documentation_string(config::shard_local_cfg().log_compression_type),
```

Comment:
it would be less lines of code to just directly pass in the property and remove the `make_documentation_string` method and have `add_topic_config_if_requested` just take a docstring as the new parameter. The method itself can check for `include_documentation`

Reply:
I'll give this a try but I do not think that is less lines of code because we would need to pass the docstring and the include_documentation boolean into `add_topic_config` and `add_broker_config`.

Reply:
Yeah you're right, this is less code.